examples/wordCount/application-DAG.json

```diff
@@ -0,0 +1,63 @@
+{
+  "applicationName": "WordCount",
+  "nodes": [
+    {
+      "task": "1",
+      "name": "text2lines",
+      "type": "producer",
+      "description": "Splits text into lines.",
+      "executor": "container",
+      "container": {
+        "image": "text2lines-image",
+        "resources": {
+          "cpu": "1",
+          "memory": "1GB",
+          "task-complexity": "no_of_instructions"
+        }
+      },
+      "comm-task": {
+        "payload": ""
+      },
+      "downstream": ["2"]
+    },
+    {
+      "task": "2",
+      "name": "lines2words",
+      "type": "prosumer",
+      "description": "Splits lines into words (stream 1).",
+      "executor": "container",
+      "container": {
+        "image": "lines2words-image",
+        "resources": {
+          "cpu": "1",
+          "memory": "1GB",
+          "task-complexity": "no_of_instructions"
+        }
+      },
+      "comm-task": {
+        "payload": ""
+      },
+      "upstream": ["1"],
```

Comment:
Hmm, the `upstream` field might be a bit redundant, since it corresponds with the `downstream` field of the corresponding producer task. Should we just use `downstream` or `outgoingEdges` here instead?

Reply:
```outgoingEdges``` seems confusing to me. ```downstream``` is better for me to understand.